lib/improver/tests/nbhood/vicinity/test_ProbabilityOfOccurrence.py

```diff
@@ -163,16 +162,14 @@ def test_additional_arguments(self):
         cube.add_aux_coord(AuxCoord(
             lead_times, "forecast_period", units="hours"), 1)
         weighted_mode = False
-        ens_factor = 0.9
         expected_cube_for_shape = cube.copy()
         expected_cube_for_shape = expected_cube_for_shape.collapsed(
             "realization", MEAN)
         orig_shape = expected_cube_for_shape.data.copy().shape
         result = (
             ProbabilityOfOccurrence(
                 distance, neighbourhood_method, radii, lead_times=lead_times,
-                weighted_mode=weighted_mode, ens_factor=ens_factor
-                ).process(cube))
+                weighted_mode=weighted_mode).process(cube))
         self.assertIsInstance(result, Cube)
         self.assertArrayAlmostEqual(result.data, expected)
```

Comment:
Why is this still the same result if we have lost the ens_factor? Was the test always a bit broken? Is it because we are doing an AlmostEqual and the change is small?

Reply:
I think the default for ens_factor was 1, so that would explain why there was no change?

Reply:
Also this has been changed now in master, so that file has been deleted.

Reply:
Fine.